strings/strings.json

```diff
@@ -554,7 +554,7 @@
     "zh-tw": "重新驗證 {0}"
   },
   "ResultsFilteringExpression": {
-    "en": "Results filtering expressions",
+    "en": "Result filtering expressions",
     "fr": "Expression de filtrage des résultats",
```

Comment:
Should the french translation use the plural `Expressions`?

Reply:
Yes, thank you!